public/scripts/models/StoryModel.js

```diff
@@ -8,7 +8,8 @@ export default Backbone.Model.extend({
 		title: '',
 		author: '',
 		category: '',
-		coverImage: ''
+		coverImage: '',
+		description: ''
 	},
 	urlRoot: '/api/v1/story',
 	idAttribute: 'id'
```

Comment:
The StoryModel shouldn't have changed for this ticket, please undo these changes.